proto/cosmos/bank/v1beta1/query.proto

```diff
@@ -41,6 +41,18 @@ service Query {
     option (google.api.http).get               = "/cosmos/bank/v1beta1/spendable_balances/{address}";
   }
 
+  // SpendableBalanceByDenom queries the spendable balance of a single denom for
+  // a single account.
+  //
+  // When called from another module, this query might consume a high amount of
+  // gas if the pagination field is incorrectly set.
+  //
+  // Since: cosmos-sdk 0.47
```

Comment:
Are we backporting this? If not we need to change that to `0.48`

Reply:
I don't know, I just removed those locally to avoid confusion, forgot to push. wdyt @tac0turtle @alexanderbez should we backport this?

Reply:
Yes, I think we should. Let's proceed. It's a small useful query update.

Reply:
awesome, will do